day-3/engine/number.py

```diff
@@ -0,0 +1,30 @@
+class Number():
+    def __init__(self, number, row, columns):
+        self.value = number
+        self.row = row
+        self.columns = columns
+
+    def get_adjacent_squares(self):
+        UP = self.row - 1
+        DOWN = self.row + 1
+        LEFT = self.columns[0] - 1
+        RIGHT = self.columns[-1] + 1
```

Comment:
Why are these variables declared as constants?
```suggestion
        up = self.row - 1
        down = self.row + 1
        left = self.columns[0] - 1
        right = self.columns[-1] + 1
```

Reply:
IDK.